R/table_utils.R

```diff
@@ -195,6 +195,11 @@ read_table_timepoint <- function(
     format = "ARROW_STREAM"
   )
 
+  if (query_result$manifest$total_chunk_count == 0) {
+    return(DBI::dbGetQuery(connector_object$conn, sql_statement))
+    zephyr::msg_success("Table read successfully!")
```

Comment:
small comment, but because the return is before the message, we will never receive the message ^^